psi_cardinality/cpp/psi_cardinality_server.cpp

```diff
@@ -131,7 +131,9 @@ StatusOr<std::string> PSICardinalityServer::ProcessRequest(
 }
 
 std::string PSICardinalityServer::GetPrivateKeyBytes() const {
-  return ec_cipher_->GetPrivateKeyBytes();
+  std::string key = ec_cipher_->GetPrivateKeyBytes();
+  key.insert(key.begin(), 32 - key.length(), '0');
```

Comment:
`'\0'` instead of `'0'`.
Maybe we should also add a test to check that servers with keys `"asdf"` and `"\0asdf"` are in fact equivalent.

Reply:
Will modify the test.